build.gradle.kts

```diff
@@ -16,6 +16,10 @@ dependencies {
     testImplementation("io.kotest:kotest-assertions-core")
 }
 
+kotlin {
```

Comment:
Why? Is it to help get green CI upon regenerating dist/github-actions-typing/lib/github-actions-typing.jar? If so, there's a workflow that generates all files in "dist".

Reply:
I'm not fully sure what you mean.
After doing the proper changes, CI failed for changed `IntegerKt.class` in `dist`.
Building the jar and putting it to `dist` changed it to complain about all class files being different.
Use the toolchain here ensures that local and CI both build with / for Java 21 so that the same class files are generated and thus the check succeeds.

Reply:
Yeah, I just meant that you didn't have to regenerate locally 😊 but what you did is also fine.